bliss/encoder/image_normalizer.py

```diff
@@ -1,6 +1,10 @@
+# flake8: noqa: WPS348
```

Comment:
WPS348 is a helpful convention. The black formatter has its moments, but for long lines, it's better to fix them manually.

Reply:
delete noqa: WPS348